processor/src/lib.rs

```diff
@@ -68,6 +61,14 @@ pub mod math {
     pub use winter_prover::math::fft;
 }
 
+pub mod crypto {
+    pub use vm_core::crypto::{
+        hash::{Blake3_192, Blake3_256, ElementHasher, Hasher, Rpo256},
+        merkle::MerkleError,
```

Comment:
Should we also re-export `MerkleStore` as it is not a requirement to create an advice provider?